docs/changelog.md

```diff
@@ -1,4 +1,8 @@
 # TBD
+### Breaking Changes
+* Split method `NetworkContext.WaitForAvailability` to `NetworkContext.WaitForAvailabilityHttpGet` and `NetworkContext.WaitForAvailabilityHttpPost`
```

Comment:
Comment 1: technically this is `WaitForEndpointAvailability` (this update missing the `Endpoint`)

Comment 2: Can we flip the ordering here, to be `WaitForHttpGetEndpointAvailability` and `WaitForHttpPostEndpointAvailability`? It'll read a bit more naturally

Reply:
Done!